src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMDecoy.h

```diff
@@ -135,7 +135,7 @@ namespace OpenMS
       @brief Selects a decoy ion from a set of ions.
     */
     std::pair<String, double> getDecoyIon(String ionid,
-                                              boost::unordered_map<String, boost::unordered_map<String, double> >& decoy_ionseries);
```

Comment:
is this inner `boost::unordered_map<String, double> >` just a `MRMIonSeries::IonSeries` or something else? if so, maybe this would make the whole syntax easier to read ... 
also present in other locations throughout the code...